_posts/2022-03-25-annotated-s4.md

```diff
@@ -0,0 +1,1728 @@
+---
+ layout: post
+ title: The Annotated S4
+ authors: Alexander M. Rush; Sidd Karamcheti
+ tags: [recent, sequence-modeling, generative-models]
+---
+
```

Comment:
Could you add either a quick summary or abstract either as the first paragraph, or as an additional `excerpt` field in the top preamble e.g.: 

```
---
 layout: post
 title: The Annotated S4
 authors: Alexander M. Rush; Sidd Karamcheti
 tags: [recent, sequence-modeling, generative-models]
 excerpt: "An annotated implementation of a series of papers developing state-space models for very long-term sequence modeling. Covers 'HiPPO: Recurrent Memory with Optimal Polynomial Projections' and ends with 'Efficiently Modeling Long Sequences with Structured State Spaces'. "
---
```

(the content is up to you but the original abstract is also totally fine)

This would make it look like the following in the `blog/` endpoint:
![image](https://user-images.githubusercontent.com/19653980/163068152-0cd5cc7f-c7ea-4f6e-8125-341049769947.png)